DigitalLearningSolutions.Web/Controllers/MyAccountController.cs

```diff
@@ -83,7 +102,21 @@ private IActionResult EditDetailsPostSave(EditDetailsViewModel model)
             var userAdminId = User.GetAdminId();
             var userDelegateId = User.GetNullableCandidateId();
 
-            if (!userService.TryUpdateUserAccountDetails(userAdminId, userDelegateId, model.Password,  model.FirstName, model.LastName, model.Email, model.ProfileImage))
+            if (!userService.TryUpdateUserAccountDetails(userAdminId,
+                userDelegateId,
+                model.Password!,
+                model.FirstName!,
+                model.LastName!,
+                model.Email!,
+                model.ProfileImage,
+                User.GetCentreId(),
+                model.JobGroupId,
+                model.Answer1,
+                model.Answer2,
+                model.Answer3,
+                model.Answer4,
+                model.Answer5,
+                model.Answer6))

```

Comment:
This is getting pretty long and hard to read now. Could we create a class for holding this information that TryUpdateUserAccountDetails is called with?